### PR TITLE
Eg g UI brain atlas final review

### PR DESCRIPTION
**Pull request by egolol**
Please test

### CODE REVIEW

**Review by annacanal — APPROVED** [review]

**Review by giovannivolpe — CHANGES_REQUESTED** [review]
use request close

**Comment by egolol** [comment]
@giovannivolpe the figure of the surf is not a Plot child so it does not have closerequest, that why I use a listener.

File: braph2genesis/genesis/create_Element.m [hunk 1]
```diff
@@ -1626,15 +1626,15 @@ function generate_gui()
                 g(2, 'end')
             end
             if  any(cellfun(@(x) ~isempty(x), gui_export))
-                g(2, ['function ui_menu_export = getGUIMenuExport(' moniker ', f)'])
```
Comment (giovannivolpe) [inline braph2genesis/genesis/create_Element.m:1629]: use moniker

File: braph2genesis/src/atlas/_PPBrainAtlasSurf.gen.m [hunk 2]
```diff
@@ -70,15 +76,15 @@ function cb_pushbutton_brain_atlas(~, ~)
...
-                y2 = normalized(2);
-                w2 = normalized(3);
-                h2 = normalized(4) /2;
+            else % golden ratio 
+                % golden ratio is defined as a+b/a = a/b = phi. phi = 1.61
+                x2 = normalized(1)+normalized(3);
+                h2 = normalized(4) / 1.61;
+                y2 = normalized(2) + h2 -.195;
+                w2 = normalized(3)*1.61;
+
```
Comment (giovannivolpe) [inline braph2genesis/src/atlas/_PPBrainAtlasSurf.gen.m:85]: spaces

File: braph2genesis/src/atlas/_PlotBrainAtlas.gen.m [hunk 3]
```diff
@@ -122,7 +122,7 @@
 %%%% ¡check_value!
 check = length(value) == 1 || length(value) == pl.get('ATLAS').get('BR_DICT').length();
 %%%% ¡default!
-1
+0
```
Comment (giovannivolpe) [inline braph2genesis/src/atlas/_PlotBrainAtlas.gen.m:125]: remove

**Review by giovannivolpe — CHANGES_REQUESTED** [review]

**Review by giovannivolpe — APPROVED** [review]

**Pull request merged**